core/Config.php

```diff
@@ -678,77 +678,39 @@ public function remove($class, $name /*,$key = null*/ /*,$value = null*/) {
 
 }
 
+class Config_LRU extends ConfigCache {
+
+	public function __construct() {
+		Deprecation::notice('3.2', ' Config_LRU is deprecated, please use ConfigCache instead', Deprecation::SCOPE_CLASS);
```

Comment:
Probably should be '4.0' due to semver.

Also, should config_lru implementation be kept in place, rather than extending the new class?